pokemon-minisite/src/app/services/login.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LoginService {
+  private isVerifiedSubject: BehaviorSubject<boolean>;
+
+  constructor() {
+    const isVerifiedLS = localStorage.getItem('isVerified') === 'true';
```

Comment:
Change to a more meaningful name
Don't use it in the constructor.
Take it to the function

Reply:
done